examples/azure/dalle.ipynb

```diff
@@ -0,0 +1,143 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Azure Dall-E image generation example\n",
+    "In this example we'll try to go over all operations needed to get image generation working using the Azure endpoints."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openai"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "For the following sections to work properly we first have to setup some things. Let's start with the `api_base`. To find your `api_base` go to https://portal.azure.com, find your resource and then under \"Resource Management\" -> \"Keys and Endpoints\" look for the \"Endpoint\" value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "openai.api_base = '' # Please add your endpoint here\n",
+    "\n",
+    "# IMPORTANT: Dall-E is currently only available through api_version '2023-04-01-preview'. This version only supports Dall-E.\n",
```

Comment:
As the email discussion shows, the version for public preview still seems to be unclear.

Reply:
Was this resolved? If so, I'm happy to merge.

Reply:
Haven't heard back. Is it ok to assume we're good here? If so, I'll merge.

Reply:
Sorry, not yet. There are still some small changes needed.

Reply:
No worries at all. I just wanted to make sure I wasn't blocking anything.